Flint/src/main/java/flint/FlintConfiguration.java

```diff
@@ -1,5 +1,25 @@
 package flint;
 
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.util.Collection;
+
+import com.github.javaparser.ast.CompilationUnit;
+
 public abstract class FlintConfiguration {
-  
+
```

Comment:
Try not to leave an extra space after the class declaration, it's readable enough without it